greenpithumb/db_store.py

```diff
@@ -110,20 +110,48 @@ def open_or_create_db(db_path):
         return _create_db(db_path)
 
 
-class DbStoreBase(object):
+class _DbStoreBase(object):
     """Base class for storing information in a database."""
 
     def __init__(self, connection):
-        """Creates a new DbStoreBase object for storing information.
+        """Creates a new _DbStoreBase object for storing information.
 
         Args:
             connection: SQLite database connection.
         """
         self._connection = connection
         self._cursor = connection.cursor()
 
+    def _do_insert(self, sql, timestamp, value):
```

Comment:
It's a pattern that I've seen a bit to try to make it obvious which function is doing the real heavy lifting. Plus, I'm nervous about having `insert` and `_insert` because I feel like it's too easy to misread/mistype one for the other.